Formula/f/foundry.rb

```diff
@@ -0,0 +1,59 @@
+class Foundry < Formula
+  desc "Blazing fast, portable and modular toolkit for Ethereum application development"
+  homepage "https://github.com/foundry-rs/foundry"
+  url "https://github.com/foundry-rs/foundry.git",
+    tag:      "v1.0.0",
+    revision: "8692e926198056d0228c1e166b1b6c34a5bed66c"
```

Comment:
```suggestion
  url "https://github.com/foundry-rs/foundry.git",
      tag:      "v1.0.0",
      revision: "8692e926198056d0228c1e166b1b6c34a5bed66c"
```
Can we use a tarball instead?